ledger/ledger-api-test-tool/README.md

```diff
@@ -25,91 +18,8 @@ into a standalone command line tool with embedded
-
-Before you can run the Ledger API test tool on your ledger, you need to load a
-specific set of DAML templates onto your ledger.
-
-1. Obtain the corresponding `.dar` file with:
-
-        ledger-api-test-tool -x
-
-2. Load the file `SemanticTests.dar` created in the current directory into your
-   Ledger.
-
-### Running the tool against a custom Ledger API endpoint
-
-Run this command to test your Ledger API endpoint exposed at host `<host>` and
-at a port `<port>`:
-
-    ledger-api-test-tool -h <host> -p <port>
-
-For example
-
-    ledger-api-test-tool -h localhost -p 6865
-
-### Exploring options the tool provides
-
-Run the tool with `--help` flag to obtain the list of options the tool provides:
-
-    $ ledger-api-test-tool --help
-    The Ledger API Test Tool is a command line tool for testing the correctness of
-    ledger implementations based on DAML and Ledger API.
-    Usage: ledger-api-test-tool [options]
-
-      --help                   prints this usage text
-      -p, --target-port <value>
-                               Ledger API server port. Defaults to 6865.
-      -h, --host <value>       Ledger API server host. Defaults to localhost.
-      --must-fail              One or more of the scenario tests must fail. Defaults to false.
-      -r, --reset              Perform a ledger reset before running the tests. Defaults to false.
-      -x, --extract            Extract the testing archive files and exit.
-
-### Try out the Ledger API Test Tool against DAML Sandbox
-
-To run the tool against [DAML
-Sandbox](https://docs.daml.com/tools/sandbox.html), run:
-
-    ledger-api-test-tool -x
-    da sandbox -- SemanticTests.dar
-    ledger-api-test-tool
-
-This should always succeed! This is useful if you do not have yet a custom
-Ledger API endpoint.
-
-The DAML Sandbox starts at by default at `localhost:6865`
-and the Ledger API Test Tool uses this as the default endpoint to test, hence
-hosts and ports command line arguments can be omitted.
-
-
-### Testing your tool from continuous integration pipelines
-
-To test your ledger in your CI pipelines, execute it as part of your pipeline:
-
-    $ ledger-api-test-tool
-    Running 10 scenarios against localhost:6865...
-    Testing scenario: Test:timeTravel
-    Testing scenario: Test:authorization_rule1_failure
-    Testing scenario: Test:authorization_success2_delegation
-    Testing scenario: Test:authorization_rule2_failure
-    Testing scenario: Test:startsAtEpoch
-    Testing scenario: Test:authorization_success1
-    Testing scenario: Test:privacy_projections1
-    Testing scenario: Test:consistency_doublespend1
-    Testing scenario: Test:consistency_doublespend3
-    Testing scenario: Test:consistency_doublespend2
-    All scenarios completed.
-    $ echo $?
-    0
-
-The tool is tailored to be used in CI pipelines: as customary, when the tests
-succeed, it will produce minimal output and return the success exit code.
+See [Ledger API Test Tool
```

Comment:
Why reference the internal location?

Reply:
This is a README for developers, which should point to the most recent docs possible, which are in the codebase.